docs/installation.rst

```diff
@@ -25,13 +25,13 @@ to view available releases. To pull the latest release, run::
 
     $ docker pull nipy/heudiconv:latest
 
-Note that when using via ``docker run`` you might need to provide your user and group IDs so they map correspondingly
+Note that when using via ``docker run``, you might need to provide your user and group IDs so they map correspondingly
 within container, i.e. like::
 
     $ docker run --user=$(id -u):$(id -g) -e "UID=$(id -u)" -e "GID=$(id -g)" --rm -t -v $PWD:$PWD nipy/heudiconv:latest [OPTIONS TO FOLLOW]
 
-`ReproIn heuristic project <https://reproin.repronim.org>`_ provides its own Docker images from
-Docker Hub `repronim/reproin` which bundle its `reproin` helper.
+Additionally, `the ReproIn heuristic project <https://reproin.repronim.org>`_ provides its own Docker images on
+Docker Hub at `repronim/reproin <https://hub.docker.com/r/repronim/reproin>`_ which bundle its `reproin` helper.
```

Comment:
The transition to talking about the ReproIn project is rather abrupt here. It would be helpful if there is some information about the relationship between HeudiConv and ReproIn before this transition.